src/GraphQL.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Netzstrategen\WooCommerceLocalStore;
+
+/**
+ * GraphQL related functionalities.
+ */
+class GraphQL {
+
+  /**
+   * @implements graphql_register_types
+   */
+  public static function graphql_register_types() {
+    self::registerLocalStockStatusEnum();
+    self::registerLocalStockObjectType();
+    self::registerLocalStockField('product');
```

Comment:
Should this also be capitalized?

Reply:
I am keeping the same function name as the filter:  `@implements graphql_register_types`

The rest of the plugin doing more or less the same. Is this what you mean?

Reply:
Oh sorry, my question should have been more clear - I meant the GraphQL schema name; the other one is starting with a capitalized letter - but maybe this inconsistency comes from somewhere else?

```suggestion
    self::registerLocalStockField('Product');
```

Reply:
You are right:

https://github.com/wp-graphql/wp-graphql-woocommerce/blob/develop/includes/type/interface/class-product.php#L32

I am surprised it worked with lowercase.